packages/flutter_tools/test/integration.shard/flutter_build_with_compilation_error_test.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/base/io.dart';
+
+import '../src/common.dart';
+import 'test_utils.dart';
+
+void main() {
+  Directory tempDir;
+  Directory projectRoot;
+  String flutterBin;
+  final List<String> targetPlatforms = <String>[
+    'apk',
+    'web',
+    if (platform.isWindows)
```

Comment:
I was hitting `cmake` failures building for linux, skipping that target for now.

Reply:
I think we just landed a revert that should fix that

Reply:
Lemme try it again then.

Reply:
```
07:35 +25 ~7 -1: test/integration.shard/flutter_build_with_compilation_error_test.dart: flutter build linux shows dart compilation error in non-verbose [E]                                            
  Expected: contains 'A value of type \'String\' can\'t be assigned to a variable of type \'int\'.'
    Actual: 'cmake not found. Run \'flutter doctor\' for more information.\n'
              ''
  
  package:test_api                                                            expect
  test/integration.shard/flutter_build_with_compilation_error_test.dart 71:7  main.<fn>
  dart:async                                                                  runZoned
  test/src/common.dart 233:14                                                 testWithoutContext.<fn>
  test/src/common.dart 232:18                                                 testWithoutContext.<fn>
  test/src/common.dart 203:18                                                 test.<fn>
  test/src/common.dart 199:5                                                  test.<fn>
```
I guess `cmake` isn't in the PATH.  I'm going to re-skip linux and investigate this later.

Reply:
SG, thank you for checking!